### ISSUE TITLE
Requesting for Filipino Translation

### ISSUE THREAD

**Issue by vinzruzell** [RefugeRestrooms/refugerestrooms#451]
Hi I want to contribute for Filipino Translation.

**Comment by vinzruzell** [comment]
Thank you for the response.

**Comment by mi-wood** [comment]
Thanks @vinzruzell for volunteering!

@DeeDeeG do you want to add the translations steps to the wiki? Even what you wrote in that comment should work :)

**Comment by vinzruzell** [comment]
Yes much better

**Comment by DeeDeeG** [comment]
Okay, here is the Wiki page: https://github.com/RefugeRestrooms/refugerestrooms/wiki/How-do-I-translate-Refuge-Restrooms.org%3F

I hope this helps.

**Comment by vinzruzell** [comment]
By the way.I have already created a new branch which is Filipino-Base

**Comment by vinzruzell** [comment]
Hi is it correct? Ive made anothet branch. Do i need to make another branch or just to commit into the master? Thank you.

**Comment by mi-wood** [comment]
Hi @vinzruzell, you can create a pull request into the main refugerestrooms/develop branch. If you're not familiar with how to do that, there is documentation here: https://help.github.com/articles/creating-a-pull-request/. 

Let me know if you need any help!

**Comment by vinzruzell** [comment]
Thank you for the information its a great help.

**Comment by vinzruzell** [comment]
Hi. I have submitted a new pull request. Which requires review...pls review
my translation..thank you.

**Comment by vinzruzell** [comment]
I would like to ask for help..because there is an error in my pull request.
Thank you.

**Comment by DeeDeeG** [comment]
Hi @vinzruzell, we are still interested in accepting more translations, if you are still interested.

**Comment by DeeDeeG** [comment]
On another subject: 

Apparently Firefox doesn't work with the `fil` (Filipino) locale, but it does work with `tl` (Tagalog). 

On the other hand, Chrome supports `fil`. So it is a good idea for us to support both `fil` and `tl`. I recommend when we have a complete Filipino translation, we copy the `fil` files into a `tl` folder and change the filenames from `.fil.yml` -> to `tl.yml`, and also change the top line for the `tl` files from `fil:` -> to `tl:`

(If I am not mistaken, Tagalog and Filipino are 99% the same thing? This is according to https://en.wikipedia.org/wiki/Filipino_language. (This seems kind of like how "Mandarin Chinese" is the standard dialect of Chinese, even though there are a lot of other regional dialects. In that analogy, Filipino is the standardized dialect of Tagalog.))

**Comment by vinzruzell** [comment]
Thank you for that.

**Comment by hnarasaki** [comment]
I would like to help with this.

**Comment by DeeDeeG** [comment]
Hello @hnarasaki, we would be happy to have a Filipino translation.

If you have any questions, feel free to ask! Would be glad to help. I see you are already started, so I hope it goes smoothly, and good luck!

**Comment by DeeDeeG** [comment]
For `devise.en.yml`, some translations can be copied from here: https://github.com/tigrish/devise-i18n/blob/master/rails/locales/tl.yml

**Comment by DeeDeeG** [comment]
Here is the updated branch with your pull requests added in: https://github.com/RefugeRestrooms/refugerestrooms/tree/Filipino-Staging-Rebase

**Comment by hnarasaki** [comment]
Progress Update: I translated a file and made a commit https://github.com/RefugeRestrooms/refugerestrooms/compare/Filipino-Staging-Rebase...hnarasaki:patch-12. (I was using the UI on GitHub.com) However, I just downloaded the GitHub for Desktop app. My plan is to make all the changes at once and then 1 PR when I'm done.

**Comment by DeeDeeG** [comment]
Thank you for the update!

**Comment by hnarasaki** [comment]
Progress Update: Still working on this. I have 5 of the 13 files translated.

**Comment by DeeDeeG** [comment]
Thank you again for the update. We definitely appreciate the work you're doing!

(I don't know if this has already been mentioned, but I wanted to mention about our process for translations: Usually, when the translations for a given language are all submitted, we try to get in contact with a second native speaker of the given language, who can then review and make any suggestions or corrections. Once that review is done (and everybody agrees on the translations being correct) then we can accept/merge the translation.

Then we would make a new release of our website when we get a chance, so that the new language is active on our real website.)

Thank you again, and I hope the rest of the process goes well.

Best regards,

\- DeeDeeG

**Comment by hnarasaki** [comment]
@DeeDeeG Great, thanks! :) Should this be added to the Wiki (https://github.com/RefugeRestrooms/refugerestrooms/wiki/How-do-I-translate-Refuge-Restrooms.org%3F) so contributors will know what to expect? "Usually, when the translations for a given language are all submitted, we try to get in contact with a second native speaker of the given language, who can then review and make any suggestions or corrections. Once that review is done (and everybody agrees on the translations being correct) then we can accept/merge the translation.

Then we would make a new release of our website when we get a chance, so that the new language is active on our real website.)"

If so, would it be possible to grant me rights to edit the Wiki?

**Comment by DeeDeeG** [comment]
> Should this be added to the Wiki
> [about how we check translations with a second native speaker]

Sure, yes. That sounds like good idea.

> If so, would it be possible to grant me rights to edit the Wiki?

Yes, I would like to do that. I will look into exactly how to do so, since it's not clear yet how I can do this. Sjould be able to figure it out in about half an hour? Hopefully?

**Comment by hnarasaki** [comment]
@DeeDeeG Cool, thanks!

**Comment by DeeDeeG** [comment]
I reached out to @mi-wood and @tkwidmer, who have permission to add you to the new "Wiki Editors" team of our GitHub organization. That should give you access to edit the wiki here.

If we don't hear back from them soon, I can also temporarily open the wiki to be editable for all GitHub users, so you can make any changes that are needed.

**Comment by DeeDeeG** [comment]
@hnarasaki I went ahead and opened up wiki edit access to all GitHub users, for now. Hopefully there won't be any vandalism?? But I don't expect any. If there is, we can fix it, so it's not a big deal anyway.

**Comment by hnarasaki** [comment]
@DeeDeeG Thanks! I edited the Wiki with a "What's Next?" section https://github.com/RefugeRestrooms/refugerestrooms/wiki/How-do-I-translate-Refuge-Restrooms.org%3F

**Comment by brynmrk** [comment]
Hi, I'd like to continue working on this. Where should I start since there is already a PR?

**Comment by DeeDeeG** [comment]
Hi @brynmrk, thanks for reaching out about this.

If you'd like to pick up where previous contributors left off, I can merge their existing work into a branch, and then you can make any changes that seem appropriate. I'll also try to point out where the existing translations are out of date with the English source files, when I get the chance.

**Comment by DeeDeeG** [comment]
Here is the updated branch: https://github.com/RefugeRestrooms/refugerestrooms/tree/Filipino-Rebase-2

**Comment by DeeDeeG** [comment]
I left a lot of comments in #556, reviewing the translations and leaving suggestions for changes. Apologies if any of them are wrong, since I don't personally know the language well.

As mentioned in [this comment](https://github.com/RefugeRestrooms/refugerestrooms/pull/556#issuecomment-448426051), there were some updates to the English files not yet reflected in the existing PRs. And since that comment, a few more (small) PRs have further changed the English source files. The relevant PRs are:

#524, #530, #560 

_(In addition to these, I have already addressed #592 and #593 in the branch that I posted in my previous comment.)_

And here is a diff of all of the relevant changes:

<details>
<summary>diff of changes to English strings (click to expand)</summary>

```diff
diff --git a/config/locales/en/restroom.en.yml b/config/locales/en/restroom.en.yml
index 8978c2b..9245be6 100644
--- a/config/locales/en/restroom.en.yml
+++ b/config/locales/en/restroom.en.yml
@@ -20,6 +20,9 @@ en:
     guess_location: 'Guess current location'
     directions: 'Directions'
     directions_hint: '(e.g.: third floor in the back, by the dressing rooms...)'
+    distance:
+      miles: miles
+      kilometers: kilometers
     comments_hint: '(e.g.: you have to be a "paying customer", just act like you are browsing for a bit...)'
     comments: 'Comments'
     accessible: 'Accessible'
@@ -46,12 +49,13 @@ en:
       downvoteerror: 'There was an unexpected problem downvoting this post.'
       downvotesuccess: 'This restroom has been downvoted! Thank you for contributing to our community.'
       new: 'A new restroom entry has been created for %{name}.'
+      edit: 'Your edit has been submitted. We will review it and update the listing'
       updated: 'This restroom entry has been updated'
       deleted: 'This restroom entry has been deleted'
       searcherror: 'There was an error searching for your location.'
       spam: 'Your submission was rejected as spam.'
-    contact:
-      contactlisting: "Contact us about this post!"
+    edit:
+      editlisting: "Propose an edit to this listing."
 
   restrooms:
     nearby:
diff --git a/config/locales/en/splash.en.yml b/config/locales/en/splash.en.yml
index c6d3df7..634d0a9 100644
--- a/config/locales/en/splash.en.yml
+++ b/config/locales/en/splash.en.yml
@@ -4,3 +4,8 @@ en:
       life-is-tough: 'Sometimes life is tough...'
       find-refuge: 'Find Refuge'
       add-restroom-button-label: 'Add A Restroom'
+
+      html-alt-attributes:
+        app-store: 'Refuge Restrooms iOS app download'
+        play-store: 'Refuge Restrooms Android app download'
+        patreon: 'Support the project on Patreon'
```

</details>

If there is anything that can be explained better, or you would like help with anything, just let us know.
